test/semotus.js

```diff
@@ -301,16 +303,20 @@ function createTemplates(objectTemplate) {
             }
         },
         validateServerCall: function () {
-            return this.canValidateServerCall;
+            console.log(failServer);
```

Comment:
Can we remove this `console.log` or add context to it?

Reply:
holdover from testing. will fix